### PR TITLE
javascript-light-bulb - Aaron Soukaphay

### PR DESCRIPTION
**Pull request by aaronsoukaphay**
Screenshot 2023-07-25 at 11 33 43 AM

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+let numberOfClicks = 0;
+
+const $lightBulb = document.querySelector('.light-bulb');
+const $row = document.querySelector('.row');
+
+$lightBulb.addEventListener('click', handleClick);
+
+function handleClick(event) {
+  numberOfClicks++;
+  console.log(numberOfClicks);
```
Comment (shawnkost) [inline javascript-light-bulb/main.js]: Remove all `console.log`s before submitting. They are only for testing / debugging purposes. Unless we specifically ask for them in the instructions

**Review by shawnkost — CHANGES_REQUESTED** [review]
One tweak here Aaron! Also please include a screenshot of the dark view as well so I can verify it looks the way it should. Or include a short gif of the functionality

File: javascript-light-bulb/main.js (hunk 1 above)
Comment (aaronsoukaphay) [reply javascript-light-bulb/main.js]: Screenshot 2023-07-25 at 11 54 25 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**